python/python/glide/async_commands/core.py

```diff
@@ -265,6 +265,26 @@ async def get(self, key: str) -> Optional[str]:
             Optional[str], await self._execute_command(RequestType.GetString, [key])
         )
 
+    async def append(self, key: str, value: str) -> int:
+        """
+        Appends a value to a key.
+        If `key` does not exist it is created and set as an empty string, so `APPEND` will be similar to SET in this special case.
+
+        See https://redis.io/commands/append for more details.
+
+        Args:
+            key (str): The key to which the value will be appended.
+            value (str): The value to append.
+
+        Returns:
+            int: The length of the string after appending the value.
+
+        Examples:
+            >>> await client.append("my_key", "world")
+                5  # Indicates that "world" has been appended to the value of "my_key", which now has a length of 5.
```

Comment:
> which now has a length of 5.

.. so the key was empty before === not existed

Please add two examples and/or fix wording